tests/test_mails.py

```diff
@@ -15,4 +17,110 @@ def test_plain_mail():
 def test_html_mail():
     mail = mails.Mail('test', subject='A test email')
     rendered = mail.html(name='World')
-    assert_equal(rendered.strip(), 'Hello <p>World</p>')
\ No newline at end of file
+    assert_equal(rendered.strip(), 'Hello <p>World</p>')
+
+class TestQueuedMail(OsfTestCase):
+    def setUp(self):
+        OsfTestCase.setUp(self)
+        self.user = factories.AuthUserFactory()
+        self.user.is_registered = True
+        self.user.save()
+
+    def test_no_login_callback_for_active_user(self):
+        mail = mails.QueuedMail()
+        mail.create(
+            to_addr=self.user.username,
+            send_at=datetime.utcnow(),
+            user=self.user,
+            mail=mails.NO_LOGIN,
+            fullname=self.user.fullname
+        )
+        self.user.date_last_login = datetime.utcnow() + timedelta(seconds=10)
+        self.user.save()
+        assert_false(mail.send_mail())
```

Comment:
Asserting this method call isn't super clear as to why the respective boolean is expected. I might think of a way to re-write this or at a minimum write docstrings for `Queuedmail#send_mail` so someone looking at these has an easy reference.